README.md

```diff
@@ -138,10 +138,10 @@ build dxvk on your local system; refer to [the dxvk README.md](https://github.co
 Building for macOS
 ---
 To build Proton for macOS, install the latest Xcode command line tools, as
-well as cmake (for openal-soft) and a recent nasm (for libjpeg-turbo), libtool, and automake. You can
+well as cmake (for openal-soft), and mingw-w64 (for dxvk) and a recent nasm (for libjpeg-turbo), libtool, and automake. You can
```

Comment:
Should be `well as cmake (for openal-soft), mingw-64 (for dxvk), and a recent nsam...`